sky/core.py

```diff
@@ -149,7 +163,10 @@ def start(
 
     Raises:
         ValueError: the specified cluster does not exist; or if ``down`` is set
-          to True but ``idle_minutes_to_autostop`` is None.
+          to True but ``idle_minutes_to_autostop`` is None; or if the specified
```

Comment:
The decision boundrary is getting rather complex here. Possible to simplify?

Reply:
Yes, agreed it is complex. We could show "argument values are invalid" instead, but I slightly prefer being more precise in this case.

Reply:
In that case, maybe split this into bullet points, i.e.

1) specified cluster does not exists

2) `idle_minutes_to_autostop` is None and `down`=True

etc.

Reply:
Good call, done.